resources/common/cloud_provider.go

```diff
@@ -62,6 +63,54 @@ var LOCATION = map[commonpb.Location]map[commonpb.CloudProvider]string{
 		AZURE: "swedencentral",
 		GCP:   "europe-north1", // Finland
 	},
+	// Germany, Frankfurt
+	commonpb.Location_EU_CENTRAL_1: {
+		AWS:   "eu-central-1",
+		AZURE: "germanywestcentral",
+		GCP:   "europe-west3",
+	},
+	// Brazil, São Paulo
+	commonpb.Location_SA_EAST_1: {
+		AWS:   "sa-east-1",
+		AZURE: "brazilsouth",
+		GCP:   "southamerica-east1",
+	},
+	// Canda, Quebec
+	commonpb.Location_CA_CENTRAL_1: {
+		AWS:   "ca-central-1",
+		AZURE: "canadaeast",
```

Comment:
since in this case azure doesn't have availability zones, I think we should change this to canadacentral (Toronto) instead since they have 3 availability zones there

Reply:
changed